src/tests/multihost/ad/test_adparameters.py

```diff
@@ -687,7 +687,6 @@ def test_0017_gssspnego_adjoin(self, multihost):
         cmd = multihost.client[0].run_command(bindRequest, raiseonerr=False)
         if cmd.returncode != 0:
             status = 'FAIL'
-            print("%s failed " % tshark_cmd)
```

Comment:
I have a better fiux for this one.

Reply:
bindRequest should be renamed to tshark_cmd  to comply with naming convention. I am not sure if the print will print anything to output and there should not be a log function instead..

Reply:
My plan is to wait until your PR is merged to rebase the fixes in this one and remove the things that have already been fixed.